resources/mock-project/pyproject.toml

```diff
@@ -2,8 +2,10 @@
 name = "mock_project"
 version = "0.0.1"
 description = ""
-dependencies = ["click==8.1.3"]
-optional-dependencies = ["black==22.8.0"]
+dependencies = ["click==8.1.3", "black==22.8.0"]
+
+[project.optional-dependencies]
+test = ["pytest>=6", "mock"]
```

Comment:
since we're still using the mock and expecting it to be a valid project, we should update these to be valid deps.

click can be a main dep. black can be an optional dep.

Reply:
unless you actually meant this https://pypi.org/project/mock/ lol

Reply:
I'm going to actually resolve this. I think I just read that as a psuedo dependency without actually thinking it through.

Reply:
Yes, I actually meant the `mock` package